src/Uno.UI.Toolkit/AssemblyInfo.cs

```diff
@@ -1,4 +1,7 @@
-﻿using System.Runtime.CompilerServices;
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+
+[assembly: AssemblyMetadata("IsTrimmable", "True")]
```

Comment:
Only net6+ will trim this assembly.
IOW The pre-net6 linker won't process this assembly.
Is that ok ?